roles/backup/defaults/main.yml

```diff
@@ -44,4 +44,7 @@ additional_labels: []
 
 # Maintain some of the recommended `app.kubernetes.io/*` labels on the resource (self)
 set_self_labels: true
+
+# Number of hours worth of events table partitions to precreate before starting backup to avoid pg_dump locks.
```

Comment:
```suggestion
# Number of whole hours worth of events table partitions to precreate before starting backup to avoid pg_dump locks.
```

Also does `precreate_partition_hours: 0` do anything bad?

Reply:
@gundalow No, in the precreate_partitions command we checks that the provided hours count is greater than 0. If it's 0 it will just skip the partition creation process.